airflow/models/__init__.py

```diff
@@ -29,6 +29,7 @@
 from airflow.models.pool import Pool
 from airflow.models.renderedtifields import RenderedTaskInstanceFields
 from airflow.models.sensorinstance import SensorInstance  # noqa: F401
+from airflow.models.session import Session
```

Comment:
Rather than doing this (and adding `airflow/models/session.py`) import it airflow.utils.db -- that's enough to get it loaded and reset and doing `reset db`